content/en/docs/Examples/services.md

```diff
@@ -0,0 +1,569 @@
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    regex_version: ^[0-9.]+$
+    web_url: https://github.com/netbox-community/netbox/releases/tag/v{{ version }}
+    icon: https://github.com/netbox-community/netbox/raw/develop/netbox/project-static/img/netbox_icon.svg
+    deployed_version:
+      url: https://netbox.example.io/
+      regex: '[0-9a-z]+ \(v([0-9.]+)\)'
+```
+
+## nextcloud/server
+Source: https://github.com/nextcloud/server
+```yaml
+service:
+  nextcloud/server:
+    type: github
+    url: nextcloud/server
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://nextcloud.com/changelog/#latest{{ version | split:"." | slice:":1" | join:""  }}
+    icon: https://github.com/nextcloud/server/raw/master/core/img/favicon.png
+    deployed_version:
+      url: https://nextcloud.example.io/status.php
+      json: versionstring
+```
+
+## opencve/opencve
+Source: https://github.com/opencve/opencve
+```yaml
+service:
+  opencve/opencve:
+    type: github
+    url: opencve/opencve
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/opencve/opencve/releases/v{{ version }}
+    icon: https://raw.githubusercontent.com/opencve/opencve/master/opencve/static/img/logo_white.png
+```
+
+## pterodactyl/panel
+Source: https://github.com/pterodactyl/panel
+```yaml
+service:
+  pterodactyl/panel:
+    type: github
+    url: pterodactyl/panel
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)
+    web_url: https://github.com/pterodactyl/panel/releases/v{{ version }}
+    icon: https://raw.githubusercontent.com/pterodactyl/panel/develop/public/assets/svgs/pterodactyl.svg
+```
+
+## pterodactyl/wings
+Source: https://github.com/pterodactyl/wings
+
+To get the version of the Wings daemon, you need the node token. You can find the token in the admin GUI in the node configuration.
+```yaml
+service:
+  pterodactyl/wings:
+    type: github
+    url: pterodactyl/wings
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)
+    web_url: https://github.com/pterodactyl/wings/releases/v{{ version }}
+    icon: https://raw.githubusercontent.com/pterodactyl/panel/develop/public/assets/svgs/pterodactyl.svg
+    deployed_version:
+      url: https://wings.example.io:8088/api/system
+      headers:
+        - key: Authorization
+          value: Bearer <API Token>
+      json: version
+```
+
+## prometheus/alertmanager
+Source: https://github.com/prometheus/alertmanager
+```yaml
+service:
+  prometheus/alertmanager:
+    type: github
+    url: prometheus/alertmanager
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/prometheus/alertmanager/blob/main/CHANGELOG.md
+    regex_content: alertmanager-{{ version }}\.linux-amd64.tar.gz
+    deployed_version:
+      url: https://alertmanager.example.io/api/v1/status
+      json: version
```

Comment:
Could you update this to `data.versionInfo.version` (AlertManager), and both Prometheus/Thanos to `data.version` please?

Reply:
Done